odim-controller/helmcharts/lenovoplugin/lenovoplugin-config.yaml

```diff
@@ -0,0 +1,12 @@
+odimra:
```

Comment:
Please remove parameters in odimra group

Reply:
Removed the odimra group